package.json

```diff
@@ -17,21 +17,24 @@
     "url": "https://github.com/SU-SWS/decanter/issues"
   },
   "dependencies": {
-    "bourbon": "^4.2.7",
+    "bourbon": "^5.0.0",
```

Comment:
These are pretty major changes. Please change the version of decanter to `3.0.0`

Reply:
Any reason why we would want to wait until Neat is updated, too? Or, would that be cause for updating to `4.0.0`?

Reply:
Neat is a pretty major update as well (By the way, i'm happy to make myself available to answer questions about updating neat. Some of the concepts on how neat handles padding, margins, and containers changed pretty significantly between neat 1.x and the latest stable release (v2.1.0).)

Reply:
Thanks, @kgcreative! Sounds like another major release with Neat would be a good idea then. So, I'll update the version now and we can again release a new version with the eventual Neat updates.

Thanks for the offer to help; I'll surely want to pick your brain on how to best make the updates.

Reply:
Neat 2.x (And the upcoming 3.x) can be used as forward-compatible fallbacks for both flexbox and grid. Had the SU Homepage Launch happened 6 months later, I would have used grid for components. CSS grid and flexbox have so many advantages over floated layouts, which will make most float-based frameworks mostly obsolete. Browser support for both are really good. Neat's breakpoint management is really great, but a lot of grid layouts can handle being responsive by using smart fractional units combinations and zero breakpoints, so even that is a massive improvement.

Reply:
Yes! We do plan on upgrading Neat as the next major undertaking (version `4.0.0`).

Shea and I have also begun to talk about (and want to meet with you, JB, about) adding CSS Grid and/or Flex Box-based classes that can be used in Decanter in addition to the Neat grid. That is, our template files for pages and other items that can and should be more rigid can leverage the current Decanter grid classes (Neat), but patterns and components can leverage more flexible ones (CSS Grid/Flex Box).

In the short-term, though, to get the project in a place where it is utilizing the latest Bourbon and Neat, I thought that this work should be prioritized first, especially before we start building out too many components and patterns that would need to be updated.